contributors/guide/github-workflow.md

```diff
@@ -93,7 +93,7 @@ Please don't use `git pull` instead of the above `fetch` / `rebase`. `git pull`
 does a merge, which leaves merge commits. These make the commit history messy
 and violate the principle that commits ought to be individually understandable
 and useful (see below). You can also consider changing your `.git/config` file via
-`git config branch.autoSetupRebase always` to change the behavior of `git pull`.
+`git config branch.autoSetupRebase always` to change the behavior of `git pull`, or another non-merge options such as `git pull --rebase`.
```

Comment:
nit: "or another non-merge option such as ..."

Reply:
Oops... I'll fix it. Thanks :+1: